linux_os/guide/services/ssh/sshd_max_sessions.var

```diff
@@ -0,0 +1,20 @@
+documentation_complete: true
```

Comment:
Please make this `var_sshd_max_sessions`

Reply:
I saw multiple examples in the repo that differed.  I didn't know which one to choose.  Resolved

Reply:
Resolved in 0c6d42e